modules/web/js/ballerina/diagram/views/default/components/utils/connector-properties.jsx

```diff
@@ -256,7 +258,18 @@ class ConnectorPropertiesForm extends React.Component {
             if (key.bType === 'string') {
                 key.value = this.addQuotationForStringValues(key.value);
             }
-            paramArray.push(key.value);
+            if (key.bType === 'enum') {
+                let enumValue = 'null';
```

Comment:
why are we putting null as a string? is it coming from the backend?

Reply:
No its not coming from the backend. Since there's no default value defined for the enum atm, the default value for the enum would be null i.e. if the user does not select any value, null would be taken as the value.